### ISSUE TITLE
Attach Radar IDs

### ISSUE THREAD

**Issue by alistra** [dear-apple/dear-apple#5]
Please attach radar issues for the problems you mention. This will make the job of people at Apple easier if they would want to fix it. And other people could duplicate the radar to make Apple prioritize it, even without knowing about this letter. 

https://blackpixel.com/writing/2012/02/radar-or-gtfo.html

**Comment by pepibumur** [comment]
Good idea @alistra . I'll update the README including them and encourage people to do the same. That'll help to have some visibility.
